File: atividade_6/posicao.c
#include <stdio.h>

int main() {
    char palavra[25], c;
    int p;

    printf("Qual a palavra:\n");
    scanf("%s", palavra);

    printf("Qual a posicao e o caracter:\n");
    scanf("%d %c", &p, &c);

    for (p; p < 25; p++) {
        if (palavra[p] == c) {
            printf("O caracter %c foi encontrado na posicao %d\n", c, p);
            break; 
        }
    }

    return 0;
}

File: atividade_6/posicao.py
palavra = input("Qual a palavra:\n")
p, c = input("Qual a posição e o caractere:\n").split()
p = int(p)

for i in range(p, len(palavra)):
    if palavra[i] == c:
        print(f"O caractere {c} foi encontrado na posição {i}")
        break
